schemas/v1.0/schema.yaml

```diff
@@ -0,0 +1,525 @@
+unevaluatedProperties: false
+$defs:
+  info:
+    $comment: https://spec.openapis.org/arazzo/v1.0.0#info-object
+    description: Provides metadata about the Arazzo description
+    type: object
+    properties:
+      title:
+        description: A human readable title of the Arazzo Description
+        type: string
+      summary:
+        description: A short summary of the Arazzo Description
+        type: string
+      description:
+        description: A description of the purpose of the workflows defined. CommonMark syntax MAY be used for rich text representation
+        type: string
+      version:
+        description: The version identifier of the Arazzo document (which is distinct from the Arazzo Specification version)
+        type: string
+    required:
+      - title
+      - version
+    $ref: '#/$defs/specification-extensions'
+    unevaluatedProperties: false
+  source-description-object:
+    $comment: https://spec.openapis.org/arazzo/v1.0.0#source-description-object
+    description: |-
+      Describes a source description (such as an OpenAPI description)
+      that will be referenced by one or more workflows described within
+      an Arazzo description
+    type: object
+    properties:
+      name:
+        description: A unique name for the source description
+        type: string
+        pattern: '^[A-Za-z0-9_\-]+$'
+      url:
+        description: A URL to a source description to be used by a workflow
+        type: string
+        format: uri-reference
+      type:
+        description: The type of source description
+        enum:
+          - arazzo
+          - openapi
+    required:
+      - name
+      - type
+    $ref: '#/$defs/specification-extensions'
+    unevaluatedProperties: false
+  workflow-object:
+    $comment: https://spec.openapis.org/arazzo/v1.0.0#workflow-object
+    description: Describes the steps to be taken across one or more APIs to achieve an objective
+    type: object
+    properties:
+      workflowId:
+        description: Unique string to represent the workflow
+        $dynamicAnchor: workflowId
+        type: string
+        pattern: '^[A-Za-z0-9_\-]+$'
+      summary:
+        description: A summary of the purpose or objective of the workflow
+        type: string
+      description:
+        description: A description of the workflow. CommonMark syntax MAY be used for rich text representation
+        type: string
+      inputs:
+        description: A JSON Schema 2020-12 object representing the input parameters used by this workflow
+        $dynamicRef: '#meta'
+      dependsOn:
+        description: A list of workflows that MUST be completed before this workflow can be processed
+        type: array
+        uniqueItems: true
+        items:
+          type: string
+      steps:
+        description: An ordered list of steps where each step represents a call to an API operation or to another workflow
+        type: array
+        uniqueItems: true
+        minItems: 1
```

Comment:
not sure if it is required to have min item = 1 
we could this is excluding people putting 
sucessActions : []  that looks valid also

Reply:
`steps` are required in the `workflows` array.  I don't think it's reasonable to assume `steps` can be empty

Reply:
sorry i miss click the location  the subject was for successAction and failureActions

i think having sucessActions : []  or failureActions : [] can be seen as a practice , some people prefer to be very explicit by putting empty array

Reply:
> i think having sucessActions : [] or failureActions : [] can be seen as a practice , some people prefer to be very explicit by putting empty array

Correct - there is no minItems requirement. I believe this has already been addressed